stable/sealed-secrets/templates/sealedsecret-crd.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
```

Comment:
I would add:

```yaml
  annotations:
    "helm.sh/resource-policy": keep
```

This way don't lose the secrets if you delete the sealed secret release.